pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1373,3 +1373,267 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func TestSlowStartBatch(t *testing.T) {
+	fn := func() error {
+		return nil
+	}
+	errFn := func() error {
+		return fmt.Errorf("fake error")
+	}
+
+	tests := []struct {
+		name                        string
+		diff                        int
```

Comment:
Call this `count` to be consistent with the function, and to stay generic - this is a test for the helper, not for RS manage replicas.